src/app/containers/FrontPageSection/index.jsx

```diff
@@ -38,10 +55,12 @@ const FrontPageSection = ({ bar, group }) => {
       </SectionLabel>
       {items.length > 1 ? (
         <StoryPromoUl>
-          {items.map(item => (
-            <StoryPromoLi key={item.id}>
-              <StoryPromo item={item} script={script} />
-            </StoryPromoLi>
+          {items.map((item, index) => (
+            <StoryPromoLiContainer
+              item={item}
+              sectionNumber={sectionNumber}
+              storyNumber={index}
+            />
           ))}
         </StoryPromoUl>
       ) : (
```

Comment:
If there is just one item, should it be considered as a topStory?

Reply:
Am not sure, better to check with product team.